Library/Homebrew/formula.rb

```diff
@@ -449,7 +449,7 @@ def installed_prefixes
   # All of current installed kegs.
   # @private
   def installed_kegs
-    installed_prefixes.map { |dir| Keg.new(dir) }
+    installed_prefixes.map { |dir| Keg.new(dir) }.sort_by { |keg| keg.version }
```

Comment:
Most uses of this are simplified if kegs are sorted by version, so I did it here. Hope it's fine?